crates/common/tedge_utils/src/notify.rs

```diff
@@ -222,6 +222,7 @@ mod tests {
         }
     }
 
+    #[cfg_attr(target_os = "macos", ignore)]
```

Comment:
Instead of ignoring every test in this module individually, why not just skip the whole test module with a `#[cfg(target_os = "linux")]` annotation? As we'd most likely wanna skip these tests on all non-Linux envs.

Reply:
This is more a todo. The notify-fs works fine on MacOS however they events manifest slightly differently here...so ideally we would invest some time in updating the tests to see why some things are different here (but this is less priority...so ignoring just shows that we have todo's rather ignoring them completely which will lead to us forgetting about them).